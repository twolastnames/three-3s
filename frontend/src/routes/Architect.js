import React from 'react';

import { Link } from 'react-router-dom';

export function Architect() {
  return (
    <div className='route'>
      <h2>Architect</h2>
      <ul>
        <li>
          <Link to="/architect/suites">Suites</Link>
        </li>
        <li>
          <Link to="/architect/create_suite">Create Suite</Link>
        </li>
      </ul>
    </div>
  );
}
