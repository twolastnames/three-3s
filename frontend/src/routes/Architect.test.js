import React from 'react';
import { shallow } from 'enzyme';
import { Architect } from './Architect';

describe('Architect', () => {
  it('renders visually', () => {
    const wrapper = shallow(<Architect />);
    expect(wrapper).toMatchSnapshot();
  });
});
