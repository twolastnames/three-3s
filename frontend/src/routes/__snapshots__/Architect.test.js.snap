// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Architect renders visually 1`] = `ShallowWrapper {}`;
